casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -536,8 +536,11 @@ class MultiParentCasperImpl[F[_]: Sync: Concurrent: Log: Time: SafetyOracle: Las
 
 object MultiParentCasperImpl {
 
-  // TODO: Extract hardcoded deployLifespanBlocks from shard config
-  val deployLifespanBlocks = 50
+  // TODO: Extract hardcoded deployLifespan from shard config
+  // Size of deploy safety range.
+  // Validators will try to put deploy in a block only for next `deployLifespan` blocks.
+  // Required to enable protection from re-submitting duplicate deploys
+  val deployLifespan = 50
```

Comment:
Why to rename it? The number represents blocks but this is not part of the name.

Reply:
https://github.com/rchain/rchain/pull/3111#discussion_r489996773
`I suggest use just deployLifespan, as the only time measure we have is number of blocks`
I mean that we don't have any other measure for block lifespan, we don't have wall clock.

Reply:
And if make it deployLifespanBlocks - it still not fully disclose how this woks, because its actually block height. Node can propose more then 50 blocks, is some of them are on the same height and orphaned.
So the concept is quite complex, we cannot express everything in name, so better make it short.

Reply:
Fair points. But I don't think is that complex. It's just an optimization to check only some deploys for signature and not all until genesis.
Property `valid after block number` should be always part of the explanation which specify the limit in which blocks deploy can be used as **valid** deploy.